### ISSUE TITLE
Broken link to LaTeX on homepage schedule

### ISSUE THREAD

**Issue by anbenson** [cmugpi/cmugpi.github.io#10]
Currently the schedule displays the link to our LaTeX content as

```
    [LaTeX][latex]
```

which seems to indicate we're missing a markdown link to

```
    https://www.cs.cmu.edu/~15131/f16/topics/latex/
```

**Comment by jez** [comment]
Oh you're right; do you want to make a PR?

**Comment by anbenson** [comment]
I don't have time tonight, but maybe later in the week? Just wanted to make an issue in case someone else was able to get to it before I could.

**Comment by anbenson** [comment]
Closed by #11
